### DISCUSSION TITLE
How to do this ['text-2xl', 'p-2'] -> ['font-size', 'padding']?

### DISCUSSION THREAD

**Discussion by ASgmbv** [dcastil/tailwind-merge#117] [Help]
Hi, I love your work on tailwind-merge, I was actually working on something similar for one of my project, but I ditched my solution for tailwind-merge because it is super nice, and I am so glad I found it. I had one question, is there any way I can make this: ['text-2xl', 'p-2', 'm-1'] -> ['font-size', 'padding', 'margin'] ?

**Comment by dcastil** [comment]
Hey @ASgmbv! 👋

Cool that you like tailwind-merge! If I understand correctly you want to get the CSS properties to a given Tailwind class, right? Unfortunately that isn't possible in tailwind-merge because that would increase the bundle size a lot and I try to keep it as small as possible.
